### PR TITLE
improve odg

### PR DESCRIPTION
**Pull request by andiwand**
besides improving odg this also contains
- support inline drawings (should effect all odf files)
- fix `<br>`
- support master page (should also effect odp)
- refactor css class translation
- apply code formatting

### CODE REVIEW

File: odf/src/ContentTranslator.cpp [hunk 1]
```diff
@@ -329,6 +327,86 @@ void TableCellTranslator(const tinyxml2::XMLElement &in, std::ostream &out,
...
 }
 
+void DrawLineTranslator(const tinyxml2::XMLElement &in, std::ostream &out,
+                        Context &context) {
+  const auto x1 = in.FindAttribute("svg:x1");
+  const auto y1 = in.FindAttribute("svg:y1");
+  const auto x2 = in.FindAttribute("svg:x2");
+  const auto y2 = in.FindAttribute("svg:y2");
+
+  if ((x1 == nullptr) || (y1 == nullptr) || (x2 == nullptr) || (y2 == nullptr))
```
Comment (TomTasche) [inline odf/src/ContentTranslator.cpp:337]: sometimes you're using braces for one-liner ifs, sometimes you don't. see https://github.com/opendocument-app/OpenDocument.core/pull/160/files?diff=unified&w=1#diff-1c9779dd88891fa701d5a9d97b8a3c59R68-R70

File: odf/src/StyleTranslator.cpp [hunk 2]
```diff
@@ -104,6 +125,7 @@ void StyleClassTranslator(const tinyxml2::XMLElement &in, std::ostream &out,
   out << "}\n";
 }
 
+// TODO
```
Comment (TomTasche) [inline odf/src/StyleTranslator.cpp:128]: create ticket and link here?

Comment (andiwand) [reply odf/src/StyleTranslator.cpp:128]: that is #35

File: odf/src/ContentTranslator.cpp (hunk 1 above)
Comment (andiwand) [reply odf/src/ContentTranslator.cpp:337]: i actually don't want braces for one-liners. it is quite hard to find them.

**Pull request merged**